CHANGES.rst

```diff
@@ -25,6 +25,6 @@
 Changes
 =======
 
-Version 1.0.0 (released 2016-01-07)
+Version 1.0.0 (released 2016-02-05)
```

Comment:
Yes, I think we talked, but never agreed. If we add the alpha release, why don't we also add release notes, and why don't we keep the each alpha version in the ``CHANGES.rst`` - i.e. why don't we treat alpha releases like any normal final release? My opinion is that we need fast turn-around time for alpha releases, and that's the reason why we should not add it. For beta release, I think we should make them completely as a normal release. Anyway, my 5 cents.